packages/react/src/components/Forms/Input/index.tsx

```diff
@@ -71,6 +71,10 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(
       setValue(props.value || "");
     }, [props.value]);
 
+    useEffect(() => {
```

Comment:
It seems that it introduced a bug, we can't reset the input when it is controlled: http://localhost:6006/?path=/story/components-forms-input--controlled

Reply:
moving this `useEffect` before the one that update value from `props.value` works.